src/editors/data/services/cms/api.ts

```diff
@@ -19,6 +14,87 @@ if (process.env.NODE_ENV === 'development' && process.env.MFE_NAME === 'frontend
   };
 }
 
+interface Pagination {
+  start: number;
+  end: number;
+  page: number;
+  pageSize: number;
+  totalCount: number;
+}
+
+interface AssetResponse {
+  assets: Record<string, string>[]; // In the raw response here, these are NOT camel-cased yet.
+}
+
```

Comment:
Most of the functions below are unchanged other than adding types - I just had to move them higher up in the file.